packages/modular-scripts/src/start.ts

```diff
@@ -6,14 +6,33 @@ import execAsync from './utils/execAsync';
 import getLocation from './utils/getLocation';
 import stageView from './utils/stageView';
 import getModularRoot from './utils/getModularRoot';
+import getWorkspaceInfo from './utils/getWorkspaceInfo';
 import { setupEnvForDirectory } from './utils/setupEnv';
 import { checkBrowsers } from './utils/checkBrowsers';
 import checkRequiredFiles from './utils/checkRequiredFiles';
 import createPaths from './utils/createPaths';
 import * as logger from './utils/logger';
 import createEsbuildBrowserslistTarget from './utils/createEsbuildBrowserslistTarget';
+import prompts from 'prompts';
+
+async function start(packageName: string): Promise<void> {
+  let target = packageName;
+
+  if (!target) {
+    const availablePackages = Object.keys(await getWorkspaceInfo());
+    const chosenTarget = await prompts<string>({
```

Comment:
@LukeSheard you mentioned we might want to abstract this in `cli`. Did you mean in `program`, since in `cli` we haven't parsed the arguments / options yet? As I see it, we should still give the possibility to specify a target app to the users (ie: `modular start myApp`) and run the interaction only if no target app is provided (`modular start`), so it should always be post-command.

Reply:
Yeah - I did mean `program` but I think for now what we have is fine - we can abstract this later if we want to re-use it.